app/renderer/actions/shared.js

```diff
@@ -22,10 +40,11 @@ export function bindClient () {
    * If we hear back with an error, reject the promise
    */
   ipcRenderer.on('appium-client-command-response-error', (evt, resp) => {
-    const {e, uuid} = resp;
+    let {e, uuid} = resp;
+    e = JSON.parse(e);
```

Comment:
does it work if an error object being serialized like this?

Reply:
The response is always a JSON object that's passed over the ipcRenderer